### PR TITLE
[IMP]hr_recruitment:change stages of job positions as per department wis...

### PR DESCRIPTION
**Pull request by tde-banana-odoo**
...e to job wise.
- Task: https://www.odoo.com/web?&debug=#id=9008&view_type=form&model=project.task&menu_id=3942&action=327
- Pad: https://pad.odoo.com/p/r.Msg2BgZQr8jUoRKF

### CODE REVIEW

File: addons/hr_recruitment/hr_recruitment.py [hunk 1]
```diff
@@ -284,30 +303,25 @@ def onchange_stage_id(self, cr, uid, ids, stage_id, context=None):
             return {'value': {'date_closed': fields.datetime.now()}}
         return {'value': {'date_closed': False}}
 
-    def stage_find(self, cr, uid, cases, section_id, domain=[], order='sequence', context=None):
+    def stage_find(self, cr, uid, applicants, job_id, domain=[], order='sequence', context=None):
         """ Override of the base.stage method
             Parameter of the stage search taken from the lead:
             - department_id: if set, stages must belong to this section or
               be a default case
         """
```
Comment (atp-odoo) [inline addons/hr_recruitment/hr_recruitment.py]: Change docstring as per change

File: addons/hr_recruitment/hr_recruitment.py [hunk 2]
```diff
@@ -573,8 +585,13 @@ def _address_get(self, cr, uid, context=None):
         user = self.pool.get('res.users').browse(cr, uid, uid, context=context)
         return user.company_id.partner_id.id
 
+    def _get_type_common(self, cr, uid, context):
+        ids = self.pool.get('hr.recruitment.stage').search(cr, uid, [('case_default','=',1)], context=context)
```
Comment (atp-odoo) [inline addons/hr_recruitment/hr_recruitment.py]: donot use pool.get()

File: addons/hr_recruitment/hr_recruitment_data.xml [hunk 3]
```diff
@@ -76,30 +76,36 @@
...
         <field name="sequence">2</field>
+        <field name="case_default" eval="True"/>
     </record>
     <record model="hr.recruitment.stage" id="stage_job3">
         <field name="name">Second Interview</field>
         <field name="sequence">3</field>
+        <field name="case_default" eval="True"/>
     </record>
     <record model="hr.recruitment.stage" id="stage_job4">
         <field name="name">Contract Proposed</field>
```
Comment (bst-odoo) [inline addons/hr_recruitment/hr_recruitment_data.xml]: Please rename "Contract Proposed" into "Contract Proposal"

Comment (msa-odoo) [inline addons/hr_recruitment/hr_recruitment_data.xml]: Ok

**Pull request closed without merging**